src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -134,6 +139,23 @@ public Visitable visit(ClassOrInterfaceDeclaration decl, Void p) {
     return result;
   }
 
+  @Override
+  public Visitable visit(ConstructorDeclaration method, Void p) {
+    String constructorMethodAsString = method.getDeclarationAsString(false, false, false);
+    String methodName =
+        this.classFQName
+            + "#"
+            + constructorMethodAsString.substring(constructorMethodAsString.indexOf(' ') + 1);
```

Comment:
Professor,

> Does this rely on the string representation having an access modifier (e.g., public)? If so, that's not okay, because no access modifier is valid Java (meaning "package private").

It does not rely on the modifier. The methodName will look something like: ```com.example.Car#Car()```. I will update the comments.